content/millennium/r4/clinical/summary/family-member-history.md

```diff
@@ -28,58 +28,47 @@ The following fields are returned if valued:
 * [Born](https://hl7.org/fhir/r4/familymemberhistory-definitions.html#FamilyMemberHistory.born_x_){:target="_blank"}
 * [Age](https://hl7.org/fhir/r4/familymemberhistory-definitions.html#FamilyMemberHistory.age_x_){:target="_blank"}
 * [Deceased](https://hl7.org/fhir/r4/familymemberhistory-definitions.html#FamilyMemberHistory.deceased_x_]){:target="_blank"}
-  * [Precision Extension](#extensions){:target="_blank"}
-* [Extension](#extensions){:target="_blank"}
-  * [Patient Adopted](#extensions){:target="_blank"}
+  * [Precision Extension](https://hl7.org/fhir/extensibility.html#Extension){:target="_blank"}
+* [Extension](https://hl7.org/fhir/extensibility.html#Extension){:target="_blank"}
+  * [Patient Adopted](https://hl7.org/fhir/extensibility.html#Extension){:target="_blank"}
```

Comment:
I can't tell if this is here
https://hl7.org/fhir/R4/extension-patient-adoptioninfo.html